### ISSUE TITLE
Unit Search missing imperil skills

### ISSUE THREAD

**Issue by Darwe-Canine** [lyrgard/ffbeEquip#185]
Currently filtering for Imperils in the Unit Search doesn't find 7* skills with imperils, or counterattack skills, such as White Knight Noel's Winter Palisade or Elfreeda's Falling Snow.

**Comment by lyrgard** [comment]
You mean on the JP part ?

**Comment by Darwe-Canine** [comment]
No, using the GL portion; filtering for Lightning or Fire imperil doesn't show Olive's Immolation Blast for example, and filtering for Cold imperil doesn't show the mentioned counterattacks.

**Comment by lyrgard** [comment]
Yes, counter attack are not supported yet. I just added cooldown skills
